libkbfs/tlf_handle_resolve.go

```diff
@@ -557,11 +674,16 @@ func parseTlfHandleLoose(
 		}
 	}
 
-	h, err := makeTlfHandleHelper(ctx, t, writers, readers, extensions)
+	h, err := makeTlfHandleHelper(
+		ctx, t, writers, readers, extensions, idGetter)
 	if err != nil {
 		return nil, err
 	}
 
+	// TODO(KBFS-2621): When we want to start creating new implicit
+	// team TLFs, and h.tlfID is null, return `iteamHandle` instead of
+	// `h`.
```

Comment:
In that case, why don't we just call the to-be-added CreateTLF RPC into service here so that at the time the handle is returned it's guaranteed to have a TLF ID?

Reply:
Yep, that's the plan!  Mentioned in the description for KBFS-2621.